### PR TITLE
Fix units

### PR DESCRIPTION
**Pull request by ericpre**
- Fix units,
- save more simulation parameters,
- scale `crop4Damax` consistently.

### CODE REVIEW

File: src/utility.cpp [hunk 1]
```diff
@@ -622,9 +622,9 @@ void setupVDOutput(Prismatic::Parameters<PRISMATIC_FLOAT_PRECISION> pars, const
 		dim2_name.write(strdatatype, dim2_name_str);
 		dim3_name.write(strdatatype, dim3_name_str);
 
-		const H5std_string dim1_unit_str("[n_m]");
-		const H5std_string dim2_unit_str("[n_m]");
-		const H5std_string dim3_unit_str("[mrad]");
+		const H5std_string dim1_unit_str("[Å]");
+		const H5std_string dim2_unit_str("[Å]");
+		const H5std_string dim3_unit_str("[rad]");
```
Comment (lerandc) [inline src/utility.cpp:627]: This looks to be related to the fact that pars.detectorAngles is in units of radians, correct? Perhaps we should just multiply the detector angles array out by 1000, I would think that more people would prefer to reference mrads.

here: https://github.com/prism-em/prismatic/blob/126dcf3f023f8bdccfa553f68bd2b7837eac5310/src/Multislice_calcOutput.cpp#L144-L154

and here: 
https://github.com/prism-em/prismatic/blob/126dcf3f023f8bdccfa553f68bd2b7837eac5310/src/PRISM03_calcOutput.cpp#L108-L127

File: src/utility.cpp [hunk 2]
```diff
@@ -1499,6 +1556,9 @@ void writeMetadata(Prismatic::Parameters<PRISMATIC_FLOAT_PRECISION> pars, double
 	H5::Attribute detector_attr = sim_params.createAttribute("d", H5::PredType::NATIVE_DOUBLE, scalar_attr);
 	H5::Attribute tx_attr = sim_params.createAttribute("tx", H5::PredType::NATIVE_DOUBLE, scalar_attr);
 	H5::Attribute ty_attr = sim_params.createAttribute("ty", H5::PredType::NATIVE_DOUBLE, scalar_attr);
+	H5::Attribute earlyStop_attr = sim_params.createAttribute("earlyCPUStopCount", H5::PredType::NATIVE_FLOAT, scalar_attr);
+	H5::Attribute rseed_attr = sim_params.createAttribute("randomSeed", H5::PredType::NATIVE_FLOAT, scalar_attr);
```
Comment (lerandc) [inline src/utility.cpp:1560]: for consistency's sake, these should be H5::PredType::NATIVE_DOUBLE

File: src/utility.cpp [hunk 3]
```diff
@@ -1499,6 +1556,9 @@ void writeMetadata(Prismatic::Parameters<PRISMATIC_FLOAT_PRECISION> pars, double
...
+	H5::Attribute crop4Da_attr = sim_params.createAttribute("crop4Damax", H5::PredType::NATIVE_FLOAT, scalar_attr);
```
Comment (lerandc) [inline src/utility.cpp:1561]: also here

File: src/utility.cpp [hunk 4]
```diff
@@ -1563,6 +1633,9 @@ void writeMetadata(Prismatic::Parameters<PRISMATIC_FLOAT_PRECISION> pars, double
 	df_attr.write(H5::PredType::NATIVE_DOUBLE, &pars.meta.probeDefocus);
 	C3_attr.write(H5::PredType::NATIVE_DOUBLE, &pars.meta.C3);
 	C5_attr.write(H5::PredType::NATIVE_DOUBLE, &pars.meta.C5);
+	earlyStop_attr.write(H5::PredType::NATIVE_FLOAT, &pars.meta.earlyCPUStopCount);
```
Comment (lerandc) [inline src/utility.cpp:1636]: and here

File: src/parseInput.cpp [hunk 5]
```diff
@@ -230,7 +230,7 @@ bool writeParamFile(Metadata<PRISMATIC_FLOAT_PRECISION> &meta,
     f << "--scan-window-xr:" << meta.scanWindowXMin_r << ' ' << meta.scanWindowXMax_r << '\n';
     f << "--scan-window-yr:" << meta.scanWindowYMin_r << ' ' << meta.scanWindowYMax_r << '\n';
     f << "--random-seed:" << meta.randomSeed << '\n';
-    f << "--4D-amax:" << meta.crop4Damax << '\n';
+    f << "--4D-amax:" << meta.crop4Damax * 1000 << '\n';
```
Comment (lerandc) [inline src/parseInput.cpp:233]: Nice catch!

**Comment by lerandc** [comment]
@ericpre Once this is settled and squared away, we should also close and reopen to merge into the dev branch instead as mentioned in prism-em/prismatic#77

File: src/utility.cpp (hunk 1 above)
Comment (ericpre) [reply src/utility.cpp]: I have changed to `pars.detectorAngles` to be in `mrad` and changed the rest accordingly.

**Comment by ericpre** [comment]
I have update the base branch to avoid having making a new PR and keep the discussion here.

**Comment by lerandc** [comment]
Thanks for the quick changes!

**Pull request merged**